vitest.config.ts

```diff
@@ -1,7 +1,13 @@
 import {readdirSync} from 'node:fs';
 import {defineConfig} from 'vitest/config';
 
-const special = ['shared', 'zero-cache', 'z2s', 'zero-pg'];
+const special = [
```

Comment:
We could probably find these by looking at the vitest.config.ts

I think that all these special ones have a `.test.workspace`.